meta-refkit-core/conf/distro/include/enable-refkit-config.inc

```diff
@@ -15,3 +15,6 @@ VIRTUAL-RUNTIME_initscripts = ""
 # defaults from OE-core.
 DISTRO_FEATURES_DEFAULT_remove = "${REFKIT_DEFAULT_DISTRO_FEATURES_REMOVE}"
 DISTRO_FEATURES ?= "${DISTRO_FEATURES_DEFAULT} ${DISTRO_FEATURES_LIBC} ${REFKIT_DEFAULT_DISTRO_FEATURES}"
+
+require conf/distro/include/usrmerge.inc
+require conf/distro/include/flatpak.inc
```

Comment:
enable-refkit-config.inc must be usable even when meta-flatpak is not used (because meta-refkit-core and thus enable-refkit-config.inc must be usable without that layer).

This is causing the "bitbake fails with missing flatpak.inc" error.

Try "include conf/distro/include/flatpak.inc" instead.